source/concurrent/SynchronizedList.ooc

```diff
@@ -0,0 +1,143 @@
+import structs/[VectorList, List]
```

Comment:
You do not need to import `VectorList`. Also, you should add `List` to `sdk.use`, and you won't need to import it anywhere either.